Fixes #1204: Release bindings and codegens to Maven Central. (#1205)
digital-asset/daml: docs/source/support/release-notes.rst

HEAD — ongoing
--------------

Java Bindings
~~~~~~~~~~~~~

> - Release the Java Bindings to the public Maven Central repository. It is recommended but not essential to remove

Reviewer: ```suggestion
- Release the Java Bindings to the public Maven Central repository. To move to using the Maven Central repository, remove
```

  the ``<repository>...</repository>`` and ``<pluginRepository>...</pluginRepository>`` blocks from Maven POM files

>   utilizing version 0.12.25 (or later) of the Java Bindings.

Reviewer: ```suggestion
  that use version 0.12.25 (or later) of the Java Bindings.
```

- Fixed a bug in an internal data structure that broke contract keys.
  See `#1623 <https://github.com/digital-asset/daml/issues/1623>`__.

Java Bindings
~~~~~~~~~~~~~

- Release the Java Bindings to the public Maven Central repository. To move to using the Maven Central repository, remove
  the ``<repository>...</repository>`` and ``<pluginRepository>...</pluginRepository>`` blocks from Maven POM files

>   that use version 0.12.26 (or later) of the Java Bindings.

Reviewer 2: This is probably going to land after https://github.com/digital-asset/daml/pull/1704 so remember to update the version number here to whatever is the next release.

Author: Updated to 0.13.1 in 7a8baf12b and will make sure it is kept up-to-date.
